app/models/grda_warehouse/hud/project.rb

```diff
@@ -206,11 +206,11 @@ class Project < Base
     end
 
     scope :confidential, -> do
-      where(confidential: true)
+      joins(:organization).where(p_t[:confidential].eq(true).or(o_t[:confidential].eq(true)))
     end
 
     scope :non_confidential, -> do
-      where(confidential: false)
+      joins(:organization).where(p_t[:confidential].eq(false).and(o_t[:confidential].eq(false)))
```

Comment:
Does this need to be a left join? In the database schema it doesn't look like the `organization` association is required on `project`, but it seems like we assume it to be present in many places in the code.

Reply:
According to the spec there should never be a project without an organization.  We've definitely seen the situation where it's missing, but we should assume it's there, and it shouldn't break anything to inner join.